pkg/health/client/modules_test.go

```diff
@@ -5,45 +5,36 @@ package client_test
 
 import (
 	"bytes"
-	"encoding/json"
-	"fmt"
 	"strings"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/assert"
 
-	"github.com/cilium/cilium/api/v1/client/daemon"
-	"github.com/cilium/cilium/api/v1/models"
 	"github.com/cilium/cilium/pkg/health/client"
-	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/healthv2/types"
 )
 
 func TestGetAndFormatModulesHealth(t *testing.T) {
 	uu := map[string]struct {
-		h client.ModulesHealth
-		e string
-		v bool
+		ss []types.Status
+		e  string
+		v  bool
 	}{
-		"empty": {
-			h: newTestMHEmpty(),
-			e: "Modules Health:\tno health payload detected",
-		},
 		"happy": {
-			h: newTestMHappy(),
-			e: "Modules Health:\tStopped(0) Degraded(1) OK(1) Unknown(0)",
+			e: "Modules Health:\tStopped(0) Degraded(2) OK(2)",
 		},
 		"happy-verbose": {
-			h: newTestMHappy(),
 			e: `Modules Health:
-agent
-├── a
-│   └── b
-│       └── c
-│           ├── fred                                            [OK] yo (20s, x1)
-│           │   └── blee                                        [OK] doh (20s, x1)
-│           └── dork                                            [DEGRADED] bozo -- BOOM! (20s, x1)
-└── m1                                                          [OK] status nominal (2s, x0)`,
+		agent
+		├── m1
```

Comment:
This might be related to how we query the db, let me take another look at this.

Reply:
Changed up how we do the remote statedb query, which will result in the status' to be sent in alphabetical order (sorted by the key).  Should the formatting code also ensure that the output is sorted? (right now it will be in the order of the status slice that is passed).